modules/statics/src/account.ts

```diff
@@ -183,6 +187,28 @@ export class HederaCoin extends AccountCoinToken {
   }
 }
 
+/**
+ * The Algo network supports tokens (assets)
+ * Algo tokens work similar to native ALGO coin, but the token name is determined by the chain,
+ * and the unique asset id on the chain: (t)algo:<assetId>
+ */
+export class AlgoCoin extends AccountCoinToken {
+  public assetURL: string;
+
+  constructor(options: AlgoCoinConstructorOptions) {
+    super({
+      ...options,
```

Comment:
why this instead of just `super(options)`?

Reply:
It can be `super(options)` as well, I just kept it consistent with all other constructors in the same file